### PR TITLE
Polish datahub

### PR DESCRIPTION
**Pull request by Brilator**

### CODE REVIEW

File: src/assets/images/datahub/arc-sharing.svg
Comment (Freymaurer) [inline src/assets/images/datahub/arc-sharing.svg:1]: Do you see this too?

image

File: src/content/docs/datahub/datahub-files.mdx [hunk 1]
```diff
@@ -1,41 +1,59 @@
 ---
-title: Working with files
+title: How to work with files in the DataHUB
```
Comment (Freymaurer) [inline src/content/docs/datahub/datahub-files.mdx:2]: Maybe rename to "Manage files", this is shown in the sidebar under "DataHUB" and we should keep titles short and descriptive

File: src/content/docs/datahub/datahub-generate-pat.mdx [hunk 2]
```diff
@@ -1,24 +1,25 @@
 ---
-title: Generate a Personal Access Token (PAT)
+title: How to generate a Personal Access Token (PAT)
```
Comment (Freymaurer) [inline src/content/docs/datahub/datahub-generate-pat.mdx:2]: "Personal Access Token", keep titles short

File: src/content/docs/datahub/datahub-group-create.mdx [hunk 3]
```diff
@@ -1,23 +1,29 @@
 ---
-title: Create a new user group
+title: How to create a new user group
```
Comment (Freymaurer) [inline src/content/docs/datahub/datahub-group-create.mdx:2]: "User Groups"

File: src/content/docs/datahub/datahub-invite-collaborators.mdx [hunk 4]
```diff
@@ -1,46 +1,49 @@
 ---
-title: Invite collaborators to your ARC
+title: How to invite collaborators to your ARC
```
Comment (Freymaurer) [inline src/content/docs/datahub/datahub-invite-collaborators.mdx:2]: Maybe "Working Together"

**Review by Freymaurer — CHANGES_REQUESTED** [review]

File: src/assets/images/datahub/arc-sharing.svg
Comment (Brilator) [reply src/assets/images/datahub/arc-sharing.svg:1]: Yep. Can't get it fixed. We'll have to stick with the png version for now.

**Comment by Brilator** [comment]
Ok, so no "How to..." to distinguish guides for specific features from base functions

File: src/content/docs/datahub/datahub-invite-collaborators.mdx (hunk 4 above)
Comment (Brilator) [reply src/content/docs/datahub/datahub-invite-collaborators.mdx:2]: Working together could mean "user groups" or "invite people". 

The datahub guides were mostly split into small single tasks. But yes, we can collect them all together again.

**Comment by Brilator** [comment]
I'll rework some of the overlapping articles ...

**Pull request merged**